docs/content/latest/explore/query-1-performance/query-tuning-intro.md

```diff
@@ -0,0 +1,380 @@
+is_over_charged | boolean                        |           |          | false
+is_paid         | boolean                        |           |          | false
+data_source     | character varying              |           |          |
+
+Indexes:
+    "contacts_pkey" PRIMARY KEY, lsm (id HASH)
+```
+
+We ran the following queries with `EXPLAIN` output to view the query execution plan generated by YSQL for a given SQL statement.
+
+```sql
+yugabyte=# explain SELECT COUNT(*) FROM contacts WHERE contacts.account_id = 1234 
+    AND contacts.is_paid = TRUE 
+    AND contacts.is_over_chared = TRUE 
+    AND (updated_at > '2021-04-12 12:00:00 '); 
+```
+
+```output
+QUERY PLAN
+----------------------------------------------
+Aggregate (cost=107.50..107.51 rows=1 width=8)
+    Seq Scan on contacts (cost=0.00..105.00 rows=1000 width=0)
+        Filter: (is_paid AND is_over_charged AND (updated_at > '2021-04-12 12:00:00'::timestamp without time zone) AND (account_id = 1234))
+(3 rows)
+```
+
+```sql
+yugabyte=# explain SELECT COUNT(*) FROM contacts WHERE contacts.account_id = 5678 
+    AND contacts.is_paid_for = TRUE 
+    AND contacts.is_over_charged = TRUE 
+    AND (updated_at > '2021-04-12 12:00:00');
+```
+
+```output
+QUERY PLAN
+----------------------------------------------
+Aggregate (cost=107.50..107.51 rows=1 width=8)
+    Seq Scan on contacts (cost=0.00..105.00 rows=1000 width=0)
+        Filter: (is_paid AND is_over_charged AND (updated_at > '2021-04-12 12:00:00'::timestamp without time zone) AND (account_id = 5678))
+(3 rows)
+```
+
+```sql
+yugabyte=# explain SELECT COUNT(*) FROM contacts WHERE contacts.account_id = 7890 
+    AND contacts.is_paid = FALSE 
+    AND (updated_at > '2021-04-12 12:00:00');
+```
+
+```output
+QUERY PLAN
+----------------------------------------------
+Aggregate (cost=107.50..107.51 rows=1 width=8)
+    Seq Scan on contacts (cost=0.00..105.00 rows=1000 width=0)
+        Filter: ((NOT is_paid) AND (updated_at > '2021-04-12 12:00:00'::timestamp without time zone) AND (account_id =1234))
+(3 rows)
+```
+
+In each case, the queries do a `Seq Scan` (sequential scan) on the tables. This operation requires scanning the entire table to retrieve the desired columns. Even though we are using the partition keys to do the look up, it still needs to do a lot of scanning. 
+
+Generally, you want to avoid `SELECT COUNT(*)` queries as they can require a full scan of the table to get the results. This can cause query degradation and in some cases cause the query to not return at all.
+
+Because most of the queries above use `account_id` as the main qualifier, we can avaoid a sequential scan by creating a direct index on that column, and then use the `INCLUDE` feature to cover the other columns that we also want in the index. Indexing is a powerful tool that can speed up queries with higher latencies. When creating an index, we want to consider the column cardinality, as well as the different index types.
```

Comment:
```suggestion
Because most of the queries above use `account_id` as the main qualifier, you can avoid a sequential scan by creating a direct index on that column, and then using the `INCLUDE` feature to cover the other columns that you also want in the index. Indexing is a powerful tool that can speed up queries with higher latencies. When creating an index, consider the column cardinality, as well as the different index types.
```